nextcloud/calendar
# Downgrade the package-lock.json format to v1

For some yet unknown reason the lockfile upgrade in https://github.com/nextcloud/calendar/pull/2986 causes build errors with npm 7.

So I downgraded my local npm, reset the package lock to the version before that PR and redid the updates. Now it builds again.

I've added node v15 to the build matrix so we hopefully catch those build breaks earlier.

This needs a manual fix for the backported updated on stable2.2.

To test

0. Make sure you have npm7
1. Run `npm ci`
2. Run `npm run dev`

on master: :boom: 
here: :sunglasses:
